src/style.css

```
@@ -0,0 +1,63 @@
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+.bigSearchBar {
+  margin: 20px auto;
+}
+
+.search_input {
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  outline: none;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  border-radius: 4px;
+  box-sizing: border-box;
+  color: #3d4e61;
+  font-family: Avenir, sans-serif;
+  font-weight: 600;
+}
+
+.bigSearch_input {
+  height: 70px;
+  padding: 0 62px;
+  font-size: 16px;
+  background: url("images/Search.svg") no-repeat 26px 25px / 19px;
+  background-size: 19px;
+  line-height: 22px;
+}
+
+.smallSearch_input {
+  height: 42px;
+  padding: 0 33px;
+  background: url("images/Search.svg") no-repeat 12px 14px / 11px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.search_input::placeholder {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
```

Placeholder inherits font-styles of it's parent element
```suggestion
.search_input::placeholder {
  font-weight: 300;
}
```